src/steps/finish/upload.tsx

```diff
@@ -159,18 +159,27 @@ type UploadFormProps = {
 };
 
 const UploadForm: React.FC<UploadFormProps> = ({ handleUpload }) => {
+  const uploadSettings = useSettings().upload ?? {};
   const {
     titleField = "required",
     presenterField = "required",
     seriesField = "optional",
-  } = useSettings().upload ?? {};
+    autofillPresenter = [],
+  } = uploadSettings;
```

Comment:
Why did you pull `uploadSettings` into a separate variable?

Reply:
Remnant of an earlier version. 🙈